scripts/validate_rst_title_capitalization.py

```diff
@@ -42,6 +42,7 @@
     "Arrow",
     "Parquet",
     "MultiIndex",
+    "MultiIndexing",
```

Comment:
I think this should just be `multi-indexing` when referenced instead

Reply:
Ok I removed it, and changed occurrences to 'Multi-indexing'